README.md

```diff
@@ -60,115 +48,38 @@
-
-#### Assets stored externally to Adobe
-This applies to assets stored outside of Adobe's Creative Cloud and accessed via preSigned URL's
+# Photoshop
 
-- Browse to https://console.adobe.io/integrations
-- Select New Integration
-- Select `Access an API`
-- Select `Photoshop`
-- Select `Service Account integration`
-- Select `Create new integration`
+## General Workflow
 
-To retrieve your access token see additional instructions at [Setting up JWT Authentication](https://www.adobe.io/authentication/auth-methods.html#!adobeio/adobeio-documentation/master/auth/JWTAuthenticationQuickStart.md)
+The typical workflow involves retrieving a PSD document manifest file via `/documentManifest` (a JSON representation of the documents layer tree), followed by one or more calls to `/renditionCreate`, `/smartObject` or `/documentOperations` to make image renditions, manipulation to your smartobject layer or optionally edit the PSD and/or create new . All endpoints are asynchronous so the response will contain the `/status` endpoint to poll for job status and results
 
-## API Keys
+### Input and Output file storage
 
-Also known as the `client_id`. You must additionally pass in your Adobe API key in the `x-api-key` header field. You’ll automatically get a developer API key when you create your Adobe I/O Console Integration.  After you've created your integration you can find your API key in the `Overview` tab of your Integration
+Clients can use assets stored on one of the following storage types:
+1. Adobe: by referencing the path to the files on Creative Cloud
+2. External: (like AWS S3) by using a presigned GET/PUT URL
+3. Azure: By generating a SAS (Shared Access Signature) for upload/download
+4. Dropbox: Generate temporary upload/download links using https://dropbox.github.io/dropbox-api-v2-explorer/
 
-## Retries
 
-- The service will retry status codes of 429, 502, 503, 504 three times.
-- You should only retry requests that have a 5xx response code. A 5xx error response indicates there was a problem processing the request on the server.
-- You should implement an exponential back-off retry strategy with 3 retry attempts.
-- You should not retry requests for any other response code.
+### SmartObject
 
-## Rate Limiting
+The Photoshop APIs currently support creating and editing of Embedded Smart Objects. Support for Linked Smart Objects is forthcoming.
 
-We have not put a throttle limit on requests to the API at this time.
+- In order to update an embedded smart object that is referenced by multiple layers you only need to update one of those layers, the effect will be reflected in all layers referencing the same smart object.
 
-# Photoshop
+- The replaced smart object takes the bounds of the new image by default. If your document contains transparent pixels (e.g some .png) , you may not get consistent bounds.
 
-## General Workflow
+The API's are documented [here](https://adobedocs.github.io/photoshop-api-docs/#api-Photoshop-document_operations)
```

Comment:
I prefer the style of putting the link inside a title or phrase. E.g., 
[API Reference for documentOperations](https://adobedocs.github.io/photoshop-api-docs/#api-Photoshop-document_operations)

I think that reads better than having a dangling *here* as a link. 

Should this be a link for smartObject?